.travis.yml

```diff
@@ -13,11 +13,7 @@ before_install:
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
   # use production url for price estimator on master or tagged
-  - export PRICE_ESTIMATOR_URL=$([[
-    $TRAVIS_BRANCH = master || \
-    $TRAVIS_BRANCH = release/* || \
-    $TRAVIS_BRANCH = hotfix/* || \
-    $TRAVIS_TAG != "" ]] && echo "production" || echo "develop")
+  - export PRICE_ESTIMATOR_URL=$([[ $TRAVIS_BRANCH = master || $TRAVIS_BRANCH = release/* || $TRAVIS_BRANCH = hotfix/* || $TRAVIS_TAG != "" ]] && echo "production" || echo "develop")
```

Comment:
Could this have happend because of yaml -> json -> text formatting?

Reply:
I wouldn't think so, cause in my computer was doing the same. However, sth should be different since this was working, so maybe u are right  🤷‍♂️